js/app/widgets/dynamicLogo.js

```diff
@@ -62,6 +63,22 @@ const DynamicLogo = new Knowledge.Class({
 
     StyleProperties: {
         /**
+         * Property: max-width
+         *
+         * -EknDynamicLogo-max-width:
+         *  * maximum width in pixels.
+         */
+        'max-width': GObject.ParamSpec.int('max-width', 'max-width', 'max-width',
```

Comment:
I share @ebassi's trepidation of adding more style properties, since it will make it that much harder to move to GTK 3.9x. I wonder if there are any plans to parse the standard CSS max-width and max-height properties upstream, even if most or all widgets do not honor them.